### DISCUSSION TITLE
Help me understand the bulk scraping GUI

### DISCUSSION THREAD

**Discussion by glottisfaun0000** [stashapp/stash#4792] [Q&A]
I often scrape like 3 or 4 scenes at a time and really don't understand how to get the scrape GUI to do what I want.

Usually what happens:
1. 'Scrape All' with a scrape endpoint
2. Metadata is populated but some results like Cover image, Title, and date are ❌'ed out and have to be individually toggled.
3. The results toggle buttons (❌⭕️✅) make a mysterious bar show up and don't change anything about what's toggled.

I can change this behavior by manually checking everything under "Results toggle ⭕️ configuration" before doing 'Scrape all' but Stash doesn't remember this settings so it requires an extra 10 clicks every time. Also "Mark as organized on save" *sometimes* is remembered.

I just find this interface really unintiutive and nothing about the toggling is described in [docs](https://docs.stashapp.cc/beginner-guides/guide-to-scraping/). I don't understand why some results are X'ed out, especially in cases where the file naming and metadata sources are basically identitcal.

The behavior I want:
1. Configure scraping to always fill every field that the results provide and isn't identical to existing (usually empty) fields.
2. Remember this setting in the future.

**Comment by philpw99** [comment]
Agree with this. The guide here
https://docs.stashapp.cc/beginner-guides/guide-to-scraping/
Only points to how to do the "Scrape All", but no more details about how to handle the results.

**Reply by glottisfaun0000** [reply]
right now i'm trying to figure out if the plugins here: https://github.com/7dJx1qP/stash-plugins?tab=readme-ov-file#stash-batch-result-toggle add or fix some of this functionality. after adding the repo and installing the XO✅ toggle still seems to do nothing to my results

**Reply by DogmaDragon** [reply]
Updated guide now explains each Tagger configuration option.
https://docs.stashapp.cc/guides/scraping-scenes-via-stash-box/

**Comment by DogmaDragon** [comment]
Tagger configuration is now saved in config.yml instead of browser in the latest Stash release, so it shouldn't be affected by clearing cache. 

As far as toggle buttons, they are 3rd plugin functionality, so I would suggest contacting the author or making the issue on their repository.

**Comment by glottisfaun0000** [comment]
The new query edit configuration panel resolved a lot of the issues I had here!
